src/main/webapp/resources/js/pages/projects/linelist/components/Table/LoadingOverlay.jsx

```diff
@@ -1,18 +1,55 @@
-import React, { Component } from "react";
+import React from "react";
+import { Progress, Space, Typography } from "antd";
+import { Component } from "react";
+import { connect } from "react-redux";
+import { grey5 } from "../../../../../styles/colors";
 
-import { Spin } from "antd";
-
-export class LoadingOverlay extends Component {
+/**
+ * React component for AG GRID to display a loading overlay
+ */
+class LoadingOverlay extends Component {
   render() {
     return (
-      <div className="ag-overlay-loading-center irida-ag-overlay">
-        <div>
-          <Spin style={{ marginRight: "1rem" }} />
-          <span className="irida-ag-overlay-loading-text">{`${i18n(
-            "linelist.agGrid.loading"
-          )}`}</span>
-        </div>
+      <div
+        style={{
+          backgroundColor: "#FFFFFF",
```

Comment:
Please make styled components for this `div` and the `Space` below

Reply:
Updated in c828433cd41c82a34d019bd2377da27bb317f436